CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [0.6] - 2017-11-10
+### Changed
+- Use authentic exceptions as defined in python-ldap instead of exceptions
+  defined internally.
```

Comment:
This should probably explain why this is useful; you can probably crib from the PR description (basically it fixed problems with trying to catch exceptions in tests).